components/org.wso2.carbon.identity.media/src/main/java/org/wso2/carbon/identity/media/file/FileBasedStorageSystemImpl.java

```diff
@@ -0,0 +1,350 @@
+/*
```

Comment:
Please note that due to a buggy behaviour in github, this file is shown as a completely added new file. To see only the actual changes, refer commit: https://github.com/wso2/identity-content-repository/pull/7/commits/f7766db8bd0d2da07ca5e025baf05fe4a1ce036b